### PR TITLE
fix: update the chain id

### PR DESCRIPTION
**Pull request by greged93**
Time spent on this PR: 0.5 day

Resolves: #NA

# Pull Request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

# What is the new behavior?
Updates the chain id, bumps Kakarot to v0.9.1 and reorders some  constants.

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: docker/hive/Dockerfile [hunk 1]
```diff
@@ -15,17 +15,17 @@ RUN apt-get update && apt-get install --no-install-recommends -y patchelf && rm
...
 #   - APIBARA_SINK_BIN_DIR: ww2ghdaw1xpx42cf93zk5n3h222438ia
 ARG APIBARA_STARKNET_BIN_DIR
 ARG APIBARA_SINK_BIN_DIR
-# Run `docker image inspect apibara/starknet:1.5.0-x86_64` to get the exact path
-# Run `docker image inspect apibara/starknet:1.5.0-aarch64` to get the exact path
+# Run `docker image inspect apibara/starknet:1.6.0-x86_64` to get the exact path
+# Run `docker image inspect apibara/starknet:1.6.0-aarch64` to get the exact path
 # ⚠️ This path is subject to change, so it's important to check it before building the image ⚠️
-COPY --from=apibara /nix/store/${APIBARA_STARKNET_BIN_DIR}-apibara-starknet-1.5.0/bin/apibara-starknet /usr/local/bin/starknet
+COPY --from=apibara /nix/store/${APIBARA_STARKNET_BIN_DIR}-apibara-starknet-1.6.0/bin/apibara-starknet /usr/local/bin/starknet
```
Comment (tcoratger) [inline docker/hive/Dockerfile:28]: We should modify `starknet:1.5.0` to `starknet:1.6.0` in docker-compose.prod.yaml, docker-compose.yaml and docker-compose.staging.yaml no?

Comment (greged93) [reply docker/hive/Dockerfile:28]: hmmmm I would rather do this in another PR where we actually check that all the dockers are ok... right now we are no stop releasing and fixing things to quickly reach testnet, once this cools down we can go over all of the dockers.

Comment (greged93) [reply docker/hive/Dockerfile:28]: I don't even know if `1.6.0` works with `docker-compose.yaml` for example

Comment (Eikix) [reply docker/hive/Dockerfile:28]: Ok let's do in subsequent PR

Comment (greged93) [reply docker/hive/Dockerfile:28]: https://github.com/kkrt-labs/kakarot-rpc/issues/1437

**Review by tcoratger — APPROVED** [review]

**Pull request merged**